lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -227,8 +231,10 @@ def __get_k8s_job_spec_template(self, ajs):
                 "containers": self.__get_k8s_containers(ajs),
                 "priorityClassName": self.runner_params['k8s_pod_priority_class'],
                 "tolerations": yaml.safe_load(self.runner_params['k8s_tolerations'] or "[]"),
-                "affinity": yaml.safe_load(self.runner_params['k8s_affinity'] or "{}"),
-                "nodeSelector": yaml.safe_load(self.runner_params['k8s_node_selector'] or "{}")
+                "affinity": yaml.safe_load(self.__get_overridable_params(ajs.job_wrapper,
```

Comment:
Are these things something we could test in https://github.com/mvdbeek/galaxy/blob/4842e0dbb9203209edcccfc72675c9b90e7b8b2d/test/integration/test_kubernetes_runner.py#L93 ? Set to some value that would prevent running the job and then override it in the destination ?

Reply:
Yeh I could try adding some tests